src/components/institution/OrganizationBox.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, BoxProps, styled, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFetchOrganization } from '../../api/hooks/useFetchOrganization';
+import { DeleteIconButton } from '../../pages/messages/components/DeleteIconButton';
+import { dataTestId } from '../../utils/dataTestIds';
+import { AffiliationSkeleton } from './AffiliationSkeleton';
+import { OrganizationHierarchy } from './OrganizationHierarchy';
+
+export const StyledOrganizationBox = styled(Box)({
+  border: '1px solid',
+  borderRadius: '4px',
+  display: 'flex',
+  padding: '0.5rem',
+  boxShadow: '0px 3px 3px 0px rgba(0, 0, 0, 0.30)',
+  backgroundColor: 'white',
+});
+
+interface OrganizationBoxProps extends Pick<BoxProps, 'sx'> {
+  unitUri: string;
+  removeAffiliation?: () => void;
+}
+
+export const OrganizationBox = ({ unitUri, sx, removeAffiliation }: OrganizationBoxProps) => {
+  const { t } = useTranslation();
+  const organizationQuery = useFetchOrganization(unitUri);
+
+  return organizationQuery.isPending ? (
+    <AffiliationSkeleton />
+  ) : organizationQuery.data ? (
+    <StyledOrganizationBox
+      sx={{
+        ...sx,
+      }}>
```

Comment:
overflødig object spread
```suggestion
      sx={sx}>
```